frontend/src/metabase/setup/actions.ts

```diff
@@ -240,15 +256,26 @@ export const setEmbeddingHomepageFlags = createAsyncThunk(
 
     if (interestedInEmbedding) {
       settingsToChange["embedding-homepage"] = "visible";
-    }
 
-    if (interestedInEmbedding && !enableEmbeddingSetByEnv) {
-      settingsToChange["enable-embedding"] = true;
-      settingsToChange["setup-embedding-autoenabled"] = true;
+      if (!isInteractiveSetByEnv) {
```

Comment:
@npretto can you help review this file and the setup tests? Here's [the spec](https://www.notion.so/metabase/Feature-flag-and-settings-for-SDK-8ed424c776194528b6332c5fc2510a0c?pvs=4#32995a32a24b4303a3d67870c1d757e2) from the project. We shouldn't be flipping embedding toggle anymore, but I'm not totally sure about the settings set by envs. This looks correct to me at glance.